### ISSUE TITLE
🐛 Bug: ctrl + f not working anymore

### ISSUE THREAD

**Issue by emre1e** [nocodb/nocodb#8489]
### Please confirm if bug report does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Open a table with a few columns and try to search for a column at the end using Ctrl + F.

Search Results: 0

When we scroll to the column, it magically appears in the search results.

### Desired Behavior

It should work like in version 205.0. Ctrl + F should allow searching for everything in the browser.

### Project Details

Node: **v20.12.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
RootDB: **pg**
PackageVersion: **0.207.1**

**Comment by dstala** [comment]
@emre1e can you share a screenshot of your issue or a video? by end of browser - do you mean last visible record on the browser? I tried it & that worked using Cmd+F

**Comment by emre1e** [comment]
![image](https://github.com/nocodb/nocodb/assets/117290601/6e7e45db-1538-4afd-91c3-637941997f24)

Now if i scroll to right, at some point he is finding it.

![image](https://github.com/nocodb/nocodb/assets/117290601/d17a8332-f453-438b-bb4d-4dc8fc1bb92e)

**Comment by emre1e** [comment]
@dstala could you reproduce the error? Its really hard to work without search :)

Thank you so much

**Comment by emre1e** [comment]
@dstala Do you have any feedback? It's really important. To search for a specific column, we have to scroll precisely to its location instead of simply using Ctrl+F.

**Comment by dstala** [comment]
@mertmit can lazy loading impact this behaviour?

**Comment by mertmit** [comment]
Yes this is related to lazy loading, we are working on a custom search to replace & recover functionality.
We will keep this issue updated. Thank you for patience @emre1e

**Comment by emre1e** [comment]
Thank you @mertmit

**Comment by emre1e** [comment]
@dstala any news on this topic. On big Tables without ctrl+f for searching, is really hard. Can wie please fix this?

**Comment by dstala** [comment]
with canvas based rendering, text are no more searchable like they were earlier on DOM based screen rendering. 

@DarkPhoenix2704 
@mertmit 

Any alternatives?

**Comment by DarkPhoenix2704** [comment]
We can check this approach https://github.com/nocodb/nocohub/issues/4428

cc: @mertmit
